src/client/utils/tracking/trackOfferEvent.ts

```diff
@@ -28,6 +28,8 @@ type OptionalParameters = {
   phoneNumberData?: GTMPhoneNumberData
   quoteCartId?: string
   memberId?: string
+  // TODO Better typedef?
+  error?: Error | any
```

Comment:
why is any needed?

Reply:
well, Error is a generic interface and catch blocks return (error: unknown) objects which are different depending on what's called inside, so I figured it's a good measure for usability. Looking forward to a better solution

Reply:
I would prefer `Error | unknown` if simply `Error` doesn't work -- to better reflect what this actually us :)

Reply:
cool, makes sense